### ISSUE TITLE
[Telemetry] Allow the changing of units beyond "compatible" ones

### ISSUE THREAD

**Issue by BladeScraper-Designs** [FrSkyRC/ETHOS-Feedback-Community#4332]
This may be a niche use-case, but for me, it's something that I think would be nice to have available.  I'm using Rotorflight with ELRS, and Rotorflight repurposes some of the CRSF (ELRS) sensors to actually be something other than the default unit suggests.  The most relevant example is GPS Altitude which is repurposed as Headspeed (RPM).  Unfortunately, while I can change the unit, I can only change it to units relevant to distance (cm, m, ft, etc).  I would like to be able to change it to RPM, but it's not available.  Maybe some way to enable extra units on a sensor for reasons like this would be useful to someone other than just me?
![20240802_210341](https://github.com/user-attachments/assets/74b82162-e296-432c-8112-574fc5e1acbc)

**Comment by bsongis-frsky** [comment]
It is annoying. It would mean that CRSF sensors would have some "protocol" unit, then "display" unit for the conversions ...

**Comment by BladeScraper-Designs** [comment]
Any update on this?
EdgeTX allows you to change the unit to any unit like mentioned.  I still have this issue, and would really like to change the "Headsped" sensor to unit "rpm", even though the original sensor is unit meters.

**Comment by rburrow87** [comment]
Yes this would be very nice to have. I currently create a calculated sensor with the display unit I want, but this would be a lot cleaner and more convenient if it's not a mess to add.

**Comment by BladeScraper-Designs** [comment]
@bsongis-frsky This is still a feature we would love to have
